brainiak/io.py

```diff
@@ -59,6 +62,10 @@ def load_images_from_dir(in_dir: Union[str, Path], suffix: str = "nii.gz",
     files = sorted(in_dir.glob("*" + suffix))
     for f in files:
         yield nib.load(str(f))
+        logger.info(
+            'file %s is read' %
```

Comment:
Right. However, `Path`s work with string formatting without explicit casting to `str`, e.g.:
```"test %s" % pathlib.Path("/")```